### PR TITLE
[Clover step-9] Make various settings for the app

### PR DESCRIPTION
**Pull request by clover-zhao**
### Summary/概要
**Description**:
- Implemented Rails i18n to commonalize Japanese and English translations for the application.
- Set the time zone to Tokyo/Japan in the Rails application to ensure correct time display.
- Configured custom error pages for 404 and 500 errors to provide a better user experience.

**Changes Made**:
- Added i18n YAML files for Japanese (ja.yml) and English (en.yml) translations.
- Modified the views to use i18n for page titles, form labels, and other text.
- Configured the application to use Tokyo/Japan as the default time zone.
- Implemented custom error pages for 404 (Not Found) and 500 (Internal Server Error) errors.
- Created appropriate view templates for the custom error pages.
- Updated routes to handle non-existing paths and display the custom 404 page.
- Ensured that the application switches to the correct locale when users use the language toggle button.
- Added a language toggle button in the application layout to allow users to switch between English and Japanese.

How to check the behavior/動作確認
- Task list
  image

- New Task
  Screenshot 2023-07-31 at 10 59 07
  Screenshot 2023-07-31 at 10 59 56
  
- Edit Task. 
  Screenshot 2023-07-31 at 10 59 24
  Screenshot 2023-07-31 at 11 00 15

- Delete Task. 
  Screenshot 2023-07-31 at 10 59 36

- Error page 404
  image

- Error page 500
  image

### CODE REVIEW

File: myapp/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+
+  # Custom method to render the 404 Not Found error page
+  def render_not_found
+    render template: 'errors/404.html', status: :not_found
+  end
+
+  # Custom method to render the 500 Internal Server Error page
+  def render_internal_server_error(exception)
+    # Log the exception for debugging purposes
+    byebug
```
Comment (leesanghun6310) [inline myapp/app/controllers/application_controller.rb]: Please delete

File: myapp/app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+    Rails.logger.error "500 Internal Server Error: #{exception.message}"
```
Comment (leesanghun6310) [inline myapp/app/controllers/application_controller.rb:32]: Please leave a logger for other error handling as well.

**Comment by leesanghun6310** [comment]
Please use I18n for the errors page and test for the errors page in each language

File: myapp/app/controllers/application_controller.rb (hunk 1 above)
Comment (clover-zhao) [reply myapp/app/controllers/application_controller.rb]: Got it.

File: myapp/app/controllers/application_controller.rb (hunk 2 above)
Comment (clover-zhao) [reply myapp/app/controllers/application_controller.rb:32]: Also add logger for other error handling

**Comment by clover-zhao** [comment]
@leesanghun6310 -san
> Please use I18n for the errors page and test for the errors page in each language

I  add I18n and system spec for error pages.
Screenshot 2023-08-01 at 13 05 58

**Review by leesanghun6310 — APPROVED** [review]
完璧に理解した

**Pull request merged**